### PR TITLE
v2.1.1

### PR DESCRIPTION
**Pull request by m-iwanicki**
Fix for unhandled flashrom parameter on NS5x_7xMU 11th gen

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -958,14 +977,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - CHANGELOG_CE.md: update date
 
 
-## [1.0.1] - 2022-09-02
```
Comment (DaniilKl) [inline CHANGELOG.md:961]: Why thi change?

Comment (m-iwanicki) [reply CHANGELOG.md:961]: This version doesn't have link (it's first tag) so it's displayed like that:

image

compare to:

image

Comment (m-iwanicki) [reply CHANGELOG.md:961]: I guess I could link to <https://github.com/Dasharo/meta-dts/compare/035081699ca860e8a315e98893830b67a817730a..v1.0.1> instead

**Comment by DaniilKl** [comment]
@m-iwanicki, looks ok, builds ok, everything is in the final image, but one test has failed.

**Review by artur-rs — APPROVED** [review]

**Pull request merged**